distributed/utils.py

```diff
@@ -417,11 +422,13 @@ def run_loop(loop=self._loop):
         if actual_thread is not thread:
             # Loop already running in other thread (user-launched)
             done_evt.wait(5)
-            if not isinstance(start_exc[0], RuntimeError):
+            if start_exc[0] is not None and not isinstance(start_exc[0], RuntimeError):
                 if not isinstance(
                     start_exc[0], Exception
                 ):  # track down infrequent error
-                    raise TypeError("not an exception", start_exc[0])
```

Comment:
I suspect this infrequent error may have been caused by this, and this 'infrequent error' may no longer occur